src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -119,684 +119,914 @@
     <value>A field with the PXDBScalar attribute must have an unbound type attribute, such as PXDate, PXDecimal</value>
+    <comment/>
   </data>
   <data name="PX1096Title" xml:space="preserve">
     <value>The signature of a method with the PXOverride attribute must match the overridden method</value>
+    <comment/>
   </data>
   <data name="PX1099Title" xml:space="preserve">
     <value>The forbidden API is used</value>
+    <comment/>
   </data>
   <data name="PX1099TitleFormat" xml:space="preserve">
     <value>The forbidden {0} "{1}" is used</value>
+    <comment/>
   </data>
   <data name="PX1099TitleFormatWithReason" xml:space="preserve">
-    <value>The forbidden {0} "{1}" is used.
-Reason: {2}</value>
+    <value>The forbidden {0} "{1}" is used.Reason: {2}</value>
+    <comment/>
   </data>
   <data name="PX1099Title_EventFormatArg" xml:space="preserve">
     <value>event</value>
+    <comment/>
   </data>
   <data name="PX1099Title_FieldFormatArg" xml:space="preserve">
     <value>field</value>
+    <comment/>
   </data>
   <data name="PX1099Title_MethodFormatArg" xml:space="preserve">
     <value>method</value>
+    <comment/>
   </data>
   <data name="PX1099Title_NamespaceFormatArg" xml:space="preserve">
     <value>namespace</value>
+    <comment/>
   </data>
   <data name="PX1099Title_PropertyFormatArg" xml:space="preserve">
     <value>property</value>
+    <comment/>
   </data>
   <data name="PX1099Title_TypeFormatArg" xml:space="preserve">
     <value>type</value>
+    <comment/>
   </data>
   <data name="SuppressDiagnosticGroupCodeActionTitle" xml:space="preserve">
     <value>Suppress the {0} diagnostic with Acuminator</value>
+    <comment/>
   </data>
   <data name="SuppressDiagnosticInSuppressionFileCodeActionTitle" xml:space="preserve">
     <value>in the Acuminator suppression file</value>
+    <comment/>
   </data>
   <data name="SuppressDiagnosticWithCommentNestedCodeActionTitle" xml:space="preserve">
     <value>in a comment</value>
+    <comment/>
   </data>
   <data name="SuppressDiagnosticWithCommentNonNestedCodeActionTitle" xml:space="preserve">
     <value>Suppress the {0} diagnostic with Acuminator in a comment</value>
+    <comment/>
+  </data>
+  <data name="PX1078MessageFormat_TypesOfDacFieldAndReferencedFieldMismatch" xml:space="preserve">
+    <value>DAC field "{0}" and referenced field "{1}" have different types. Expected type for the DAC field is {2}.</value>
+    <comment/>
+  </data>
+  <data name="PX1078MessageFormat_TypesOfDacFieldAndReferencedFieldHaveDifferentSize" xml:space="preserve">
+    <value>DAC field "{0}" and referenced field "{1}" have different length specified in data type attribute. Expected length for DAC field is {2}.</value>
+    <comment/>
+  </data>
+  <data name="PX1078Title_TypesOfDacFieldAndReferencedFieldHaveDifferentSize" xml:space="preserve">
+    <value>DAC field and field referenced with PXSelectorAttribute have different size</value>
```

Comment:
@acasj the messages without format args should be either made consistent with the messages with format args, or deleted.

Reply:
done